api/app/signals/apps/signals/migrations/0094_SIG-2170_new_categories.py

```diff
@@ -0,0 +1,208 @@
+            * Responsible department codes -> ASC, AEG, WAT
+            * Handling -> ???
+            * SLA 3 weekdagen
+    NIEUW: Oplaadpunt
+            * Parent category slug -> wegen-verkeer-straatmeubilair
+            * Responsible department codes -> VOR
+            * Handling -> ???
+            * SLA 5 weekdagen
+    NIEUW: Tijdelijke Verkeersmaatregelen
+            * Parent category slug -> wegen-verkeer-straatmeubilair
+            * Responsible department codes -> STW
+            * Handling -> HANDLING_A3DEC
+            * SLA ???
+    """
+    ServiceLevelObjective = apps.get_model('signals', 'ServiceLevelObjective')
+
+    Department = apps.get_model('signals', 'Department')
+    stw = Department.objects.get(code='STW')
+    aeg = Department.objects.get(code='AEG')
+    asc = Department.objects.get(code='ASC')
+    wat = Department.objects.get(code='WAT')
+    vor = Department.objects.get(code='VOR')
+
+    Category = apps.get_model('signals', 'Category')
+
+    # OPENBAAR GROEN EN WATER ------------------------------------------------------------------------------------------
+
+    parent_category = Category.objects.get(slug='openbaar-groen-en-water', parent__isnull=True)
+
+    # Boom - boomstob
+    new_category = Category.objects.create(
+        parent=parent_category,
+        name='Boom - boomstob',
+        handling='I5DMC'
+    )
+
+    # SLO 5 Werkdagen
+    ServiceLevelObjective.objects.create(category=new_category, n_days=5, use_calendar_days=False)
+
+    # All departments can view
+    new_category.departments.add(*Department.objects.exclude(code='STW'),
+                                 through_defaults={'is_responsible': False, 'can_view': True})
+    # STW is responsible
+    new_category.departments.add(stw, through_defaults={'is_responsible': True, 'can_view': True})
+
+    # AFVAL ------------------------------------------------------------------------------------------------------------
+
+    parent_category = Category.objects.get(slug='afval', parent__isnull=True)
+
+    # Container glas kapot
+    new_category = Category.objects.create(
+        parent=parent_category,
+        name='Container glas kapot',
+        handling='A3WMC'
+    )
+
+    # SLO 3 weekdays
+    ServiceLevelObjective.objects.create(category=new_category, n_days=3, use_calendar_days=True)
+
+    # All departments can view
+    new_category.departments.add(*Department.objects.exclude(code='AEG'),
+                                 through_defaults={'is_responsible': False, 'can_view': True})
+    # AEG is responsible
+    new_category.departments.add(aeg, through_defaults={'is_responsible': True, 'can_view': True})
+
+    # Container glas vol
+    new_category = Category.objects.create(
+        parent=parent_category,
+        name='Container glas vol',
+        handling='A3WMC'
+    )
+
+    # SLO 3 weekdays
+    ServiceLevelObjective.objects.create(category=new_category, n_days=3, use_calendar_days=True)
```

Comment:
see above (should be 3 working days) (according to ticket)

Reply:
Fixed